requirements/requirements.txt

```diff
@@ -95,3 +95,5 @@ github3.py==1.0.0a4
 django-bulk-update==1.1.10
 csiphash==0.0.4
 json-delta==2.0
+email_validator==1.0.1
```

Comment:
@benrudolph Any reason you went with this over the built in django validator? https://docs.djangoproject.com/en/1.8/ref/validators/#validate-email